Package.swift

```diff
@@ -16,14 +16,16 @@ let package = Package(
     .library(name: "HtmlTestSupport", targets: ["HtmlTestSupport"]),
     .library(name: "HtmlPrettyPrint", targets: ["HtmlPrettyPrint"]),
     .library(name: "HttpPipeline", targets: ["HttpPipeline"]),
+//    .executable(name: "HttpPipelineExample", targets: ["HttpPipelineExample"]),
```

Comment:
do we wanna leave these in?

Reply:
I'd like to get it workin but didn't want it to block this PR.